app/controllers/results_controller.rb

```diff
@@ -1,17 +1,17 @@
 class ResultsController < ApplicationController
+  before_action :find_assessment
```

Comment:
What do you think about calling it from the body of each action instead? It's less dry, but I tend to like seeing the instance variables that I intend to use in the view defined entirely in the controller actions.

so it'd be:

```ruby
def new
  @assessment = find_assessment
  #...
end

def create
  @assessment = find_assessment
  #...
end
```

I can go either way on this, particularly in a small controller like this.